cvat/apps/iam/signals.py

```diff
@@ -42,6 +42,9 @@ def create_user(sender, user=None, ldap_user=None, **kwargs):
                     if role == settings.IAM_ADMIN_ROLE:
                         user.is_staff = user.is_superuser = True
                     break
+        # add default group if no other group has been assigned and the user is not a superuser
+        if len(user_groups) == 0 and not (user.is_superuser or user.is_staff):
+            user_groups.append(Group.objects.get(name=settings.IAM_DEFAULT_ROLE))
```

Comment:
```suggestion
        if not len(user_groups):
            user_groups.append(Group.objects.get(name=settings.IAM_DEFAULT_ROLE))
```
Why do we need to check here whether a user is a superuser?
If I remember correctly, `settings.IAM_TYPE == 'LDAP'` does not assume a superuser to be created, e.g. via python manage.py createsuperuser. In this case, if a user is created with `is_staff`/`is_superuser`==True, the `user_groups` will already include `IAM_ADMIN_ROLE`.